notebooks/stress-tests.ipynb

```diff
@@ -105,7 +105,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 29,
+   "execution_count": 34,
```

Comment:
FYI @ahandan this commit f23ef78cb169c7372b2704b042f943a747e11fb7 "Specify the timeout is Client side" did not actually contain any code change.

Reply:
Nice Catch !